Makefile

```diff
@@ -18,6 +19,20 @@ setup:
 	vagrant up
 	vagrant ssh-config > ssh_config
 
+
+batch:
+	$(foreach GROUP,$(GROUPS),GROUP=$(GROUP) $(MAKE) run-group || exit 1;)
+
+run-group: HOSTS=$(shell ansible ${GROUP} -i hosts --list-hosts | tail -n +2)
+run-group:
+	vagrant up ${HOSTS}
+	vagrant status ${HOSTS}
+	vagrant ssh-config ${HOSTS} >ssh_config
+	ANSIBLE_LIMIT=${GROUP} make run
+	vagrant destroy -f ${HOSTS}
+
+.PHONY: batch clean run run-elastic run-group run-oss setup ve
```

Comment:
Can you move this down to the bottom of the file. Feels like it should either be at the top or bottom of the file. Or break it down into one line `.PHONY: <target>` the preceeds each target.

```
.PHONY: run-group
run-group:
````

Reply:
I moved the phony targets to the end of the file.